packages/lib/src/core/Analytics/utils.ts

```diff
@@ -63,3 +70,133 @@ export const processAnalyticsData = (analyticsData: AnalyticsData): AnalyticsDat
         return acc;
     }, {});
 };
+
+export const getCardConfigData = (cardProps: CardElementProps): CardConfigData => {
+    // Extract props from cardProps - mostly setting a default value, if prop not found
+    const {
+        autoFocus = CardDefaultProps.autoFocus,
```

Comment:
Why not using the `cardProps` directly? I believe the Card element should already merged the default values with the user defined value (if provided)?

Reply:
But all the default values haven't been set in the `Card` element. 
The default values you see being used here are actually being pulled from the `CardInput` component and are "filling in" the gaps of any missing config

Reply:
This doesn't work out the box... but I can _make_ it work